src/services/expense.service.js

```
@@ -0,0 +1,63 @@
+const { v4: uuidv4 } = require('uuid');
+const { Op } = require('sequelize');
+const { Expense } = require('../models/models');
+
+async function getAll(userId, categories, dateFrom, dateTo) {
+  return Expense.findAll({
+    where: {
+      userId: userId,
+      categories: categories,
+      date: {
+        [Op.gte]: dateFrom,
+        [Op.lte]: dateTo,
+      },
+    },
+  });
+}
```

Add basic error handling (try/catch) for async DB operations or let errors propagate to a centralized error handler via `next(err)`. Right now any DB error will cause unhandled promise rejections or crash the request. Consider wrapping each handler call site with try/catch in controllers or return Promises that the framework can handle safely.